Automattic/wp-calypso
# Moves the G Suite add user form

#### Changes proposed in this Pull Request

* Moves Add user component to the G Suite folder
* Moves css into the component

#### Testing instructions

Non-primary site with one custom domain no G Suite:

* Goto Domains
* Goto Email tab
* Observe G Suite Marketing copy
* Click Add G Suite button

Non-primary site with one custom domain no G Suite:

* Goto Domains
* Click into custom domain
* Click email menu item
* Click Add G Suite button
* Follow What to test from below

Site with two custom domains and non-primary site with no G Suite:

* Goto Domains
* Click into non-primary custom domain
* Click email menu item
* Click Add G Suite button
* Follow What to test from below

Site with G Suite

* Goto Domains
* Click Email tab
* Click Add G Suite User
* Follow What to test from below

What to test:

* Observe domain in header
* Fill out form, continue to checkout, observe domain matches selected domain
* Observe domain in select should match header
* Fill out form, change select, continue to checkout, observe domain matches selected domain
* If user has name observe it auto-fills
* Observe that errors appear if username is invalid or names are blank
* Click `Add another`, do new fields appear?
* If you fill out fields for one user and continue to checkout, does it work?
* If you fill out fields for two users and continue to checkout, does it work?
* If you fill out fields for two users with different domains and continue to checkout, does it work?
